privacy-snapshot.json

```diff
@@ -29,6 +29,8 @@
   "github.com",
   "goerli.infura.io",
   "lattice.gridplus.io",
+  "linea-mainnet.infura.io",
```

Comment:
Can these be removed now that more stuff is mocked?

Reply:
I added mocks for the test that was failing `test/e2e/tests/settings/account-token-list.spec.js` 
But I still had to add `sepolia.infura.io` to privacy snapshot